dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/validation/field/validators/OrcidFieldValidatorTest.java

```diff
@@ -75,9 +75,6 @@ public void validateValue__non_matching_identifier_type() {
         FieldValidationResult result = validator.validateValue(isni, identifier, params, Collections.emptyMap());
 
         // then
-        assertThat(result.isOk()).isTrue();
-        assertThat(result.getField()).isNull();
-        assertThat(result.getMessage()).isEmpty();
```

Comment:
Don't forget to add those assertions back in.

Reply:
fixed